### ISSUE TITLE
Error when building maven project

### ISSUE THREAD

**Issue by bbagish** [rest-assured/rest-assured#1209]
When I try to build maven I'm getting this message: The POM for com.sun.xml.bind:jaxb-osgi:jar:2.2.10 is invalid, transitive dependencies (if any) will not be available

Rest assured versions I've tried: 4.0.0 and 4.1.0

When I remove rest assured from my pom. The warning disappear.
Last time I tried building project on Friday (8/30/19) It was working fine. I believe the new changes could lead to the issue.

**Comment by johanhaleby** [comment]
This should be fixed in version 4.1.1. Until then try version `4.1.1-SNAPSHOT` after having added the following maven repo:

```xml
<repositories>
        <repository>
            <id>sonatype</id>
            <url>https://oss.sonatype.org/content/repositories/snapshots/</url>
            <snapshots />
        </repository>
</repositories>
```

or you could excluding com.sun.xml.bind:jaxb-osgi:jar file if you're not using OSGi.

**Comment by bbagish** [comment]
@johanhaleby Thank you sir!
